src/HashBus.Projector.MostHashtagged/App.cs

```diff
@@ -1,32 +1,50 @@
 namespace HashBus.Projector.MostHashtagged
 {
+    using System;
     using System.Collections.Generic;
-    using System.Threading;
+    using System.Threading.Tasks;
+    using HashBus.Application.Events;
     using HashBus.NServiceBusConfiguration;
     using HashBus.ReadModel;
     using HashBus.ReadModel.MongoDB;
     using MongoDB.Driver;
     using NServiceBus;
+    using Hashtag = HashBus.ReadModel.Hashtag;
 
     class App
     {
-        public static void Run(string mongoConnectionString, string mongoDBDatabase, string endpointName)
+        public static async Task Run(string mongoConnectionString, string mongoDBDatabase, string endpointName)
         {
             var mongoDatabase = new MongoClient(mongoConnectionString).GetDatabase(mongoDBDatabase);
 
-            var busConfiguration = new BusConfiguration();
-            busConfiguration.EndpointName(endpointName);
-            busConfiguration.UseSerialization<JsonSerializer>();
-            busConfiguration.EnableInstallers();
-            busConfiguration.UsePersistence<InMemoryPersistence>();
-            busConfiguration.RegisterComponents(c =>
+            var endpointConfiguration = new EndpointConfiguration(endpointName);
+            endpointConfiguration.UseSerialization<JsonSerializer>();
+            endpointConfiguration.EnableInstallers();
+            endpointConfiguration.UsePersistence<InMemoryPersistence>();
+            endpointConfiguration.RegisterComponents(c =>
                 c.RegisterSingleton<IRepository<string, IEnumerable<Hashtag>>>(
                     new MongoDBListRepository<Hashtag>(mongoDatabase, "most_hashtagged__hashtags")));
-            busConfiguration.ApplyMessageConventions();
+            endpointConfiguration.ApplyMessageConventions();
+            endpointConfiguration.ApplyErrorAndAuditQueueSettings();
 
-            using (Bus.Create(busConfiguration).Start())
+            var transportExtensions = endpointConfiguration.UseTransport<MsmqTransport>();
+            var routing = transportExtensions.Routing();
+            routing.RouteToEndpoint(
+                messageType: typeof(TweetAnalyzed),
+                destination: "HashBus.Application");
+
+            var endpointInstance = await Endpoint.Start(endpointConfiguration)
+                .ConfigureAwait(false);
+            try
+            {
+                Console.WriteLine("\r\n Bus created and configured; press any key to stop program\r\n");
+                
+                Console.ReadKey();

```

Comment:
Same comment as above regarding the application.

This applies to all the projectors.